triton/tut/parallel-shared.rst

```diff
@@ -0,0 +1,257 @@
+===========================================================
+Shared memory parallelism: multithreading & multiprocessing
+===========================================================
+
+
+.. admonition:: Abstract
+
+   * Verify that your program can utilize multiple CPUs.
+
+   * Use ``--cpus-per-task=C`` to reserve multiple CPUs for your job.
```

Comment:
```suggestion
   * Use ``--cpus-per-task=C`` to reserve ``C`` CPUs for your job.
```
Or maybe "N" to be a less confusing metavariable?  For about one second when reading this, I wondered "what's this new option?"

Reply:
But I see why it's see, when N is used for number of iterations below.

Reply:
Yes. And N is used for other things in other chapters (like number of nodes).